service_quotas/service_quotas.go

```diff
@@ -26,30 +27,37 @@ type UsageCheck interface {
 	Usage() ([]QuotaUsage, error)
 }
 
-func newUsageChecks(c client.ConfigProvider, cfgs ...*aws.Config) map[string]UsageCheck {
+func newUsageChecks(c client.ConfigProvider, cfgs ...*aws.Config) (map[string]UsageCheck, []UsageCheck) {
 	// all clients that will be used by the usage checks
 	ec2Client := ec2.New(c, cfgs...)
 
-	return map[string]UsageCheck{
+	serviceQuotasUsageChecks := map[string]UsageCheck{
 		"L-0EA8095F": &RulesPerSecurityGroupUsageCheck{ec2Client},
 		"L-2AFB9258": &SecurityGroupsPerENIUsageCheck{ec2Client},
 		"L-E79EC296": &SecurityGroupsPerRegionUsageCheck{ec2Client},
 		"L-34B43A08": &StandardSpotInstanceRequestsUsageCheck{ec2Client},
 		"L-1216C47A": &RunningOnDemandStandardInstancesUsageCheck{ec2Client},
 	}
+
+	otherUsageChecks := []UsageCheck{
+		&AvailableIpsPerSubnetUsageCheck{ec2Client},
+	}
+
+	return serviceQuotasUsageChecks, otherUsageChecks
 }
 
 // QuotaUsage represents service quota usage
 type QuotaUsage struct {
-	// Name is the name of the resource, eg. the ID of the VPC or
-	// the same as the description for single-resource quotas
-	// (eg. VPCs per region)
 	// Name is the name of the quota (eg. spot_instance_requests)
+	// or the name given to the piece of exported availibility
+	// information (eg. available_IPs_per_subnet)
 	Name string
 	// ResourceName is the name of the resource in case the quota
 	// is for multiple resources. As an example for "rules per
 	// security group" the ResourceName will be the ARN of the
-	// security group
+	// security group.
+	// ResourceName, in the case of `available IPs per subnet`,
+	// is constructed as the "subnet ARN" + "|" + "CIDR block"
```

Comment:
I think it makes sense to add that comment to the specific usage check, instead of adding it here.